consensus/src/dag/types.rs

```diff
@@ -818,6 +847,41 @@ impl TryFrom<ConsensusMsg> for DAGMessage {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct DAGRpcResult(pub Result<DAGMessage, DAGRpcError>);
```

Comment:
why we need the wrapper type? because of the trait impl?

Reply:
yes, I initially had a `pub type = Result`, but rust was complaining that I cannot impl trait for Result because its in a different crate.